configuration/config2/ConfigItem.hpp

```diff
@@ -93,22 +104,38 @@ class ConfigItem : public std::enable_shared_from_this<ConfigItem>
         std::string m_displayName;
         std::string m_className;
         std::string m_category;  // used for further subdividing to the user
-		bool m_isConfigurable;
-        std::map<std::string, std::shared_ptr<ConfigItem>> m_children; 
-        std::shared_ptr<CfgValue> m_pValue;   // value for this item (think of it as the VALUE for an element <xx attr= att1=>VALUE</xx>)
-		std::map<std::string, std::shared_ptr<CfgValue>> m_attributes;   // attributes for this item (think in xml terms <m_name attr1="val" attr2="val" .../> where attrN is in this vector
+        std::string m_componentName;   
```

Comment:
extra spaces. Your environment might be lacking the git hooks Richard created which prevents these types of issues to be allowed when you commit/push.